### DISCUSSION TITLE
Speed Limit overlay idea

### DISCUSSION THREAD

**Discussion by NathanARF** [streetcomplete/StreetComplete#4923] [Ideas]
Hi,
Would it be a good idea to add a 'Speed Limit' overlay similar to the new 'Surfaces' overlay? It would come in handy not just for adding speed limits to roads where they are missing, but updating speed limit tags on roads where they may be out of date or the `maxspeed:type` tag is missing or incorrect. Also for visualisation alone it would be much more useful to see where speed limits have been mapped along ways compared to solving the individual "point" quests for a road.

Sorry if this has been mentioned before; I did a quick search and found no other discussion similar to this.

**Comment by riQQ** [comment]
See #4282

**Reply by NathanARF** [reply]
I'll take a look at that thanks. Am I searching/posting in the wrong place then?

**Reply by riQQ** [reply]
As the linked issue is open, labeled as feature and already has some information and discussion, you should add anything you have to add over there. You can also add your :+1: reaction to signal your interest in this feature.

**Reply by Helium314** [reply]
> Am I searching/posting in the wrong place then?

Searching in issues is always a good idea.
But since you didn't know about the issue, opening a discussion isn't anything wrong.
